internal/learn/about-iota/data-transfer.md

```diff
@@ -67,6 +71,12 @@ In the upcoming 2.0 IOTA version, this PoW requirement will become adaptive and
 
 ## Message Structure
 
+:::note
+
+Protocol Version and uint8 have replaced the formed NetworkID and uint64 ahead of the Stardust release.
+
+:::
+
```

Comment:
I think there is a bit of misunderstand what I mean with the "single version" of these docs and indicating what changed. I don't mean "describe what has been changed for the Stardust release", I mean we keep the main documentation reflecting the current state of mainnet and then use notes, sections, tabs, etc to document the difference in Shimmer, IOTA 2.0, etc. For example something like (just to get the idea):

```
The NetworkID is a uint64, indicating what network this message is meant for.

:::note
On Shimmer the NetworkID is renamed to Protocol Version and is now a uint8.
:::
```